source/install/desktop-app-changelog.rst

```diff
@@ -52,6 +52,7 @@ All Platforms
 - Shortened the maximum length (width) for server tab names to 224px.
 - Updated the menu bar and system tray icons for improved contrast.
 - Removed ``libappnotify1`` as a dependency requirement in Debian installers as it's no longer shipped in Debian's Bullseye. It's still recommended to install where available.
+- "Minimize to tray" setting was removed.
```

Comment:
is it minimize to tray or start minimized?

Reply:
The setting described here is "minimizeToTray" https://github.com/mattermost/desktop/issues/1687, but not sure if it's the official name.

Reply:
@Willyfrog @amyblais I'm not clear on what actually changed here.

Reply:
I'll remove it for now.